### PR TITLE
[sdg] Use unDataLabel for place names

### PR DESCRIPTION
**Pull request by chejennifer**
- do this for map, scatter, line, and bar

Screenshot 2023-09-13 at 10 32 49 AM
Screenshot 2023-09-13 at 10 33 35 AM
Screenshot 2023-09-13 at 10 40 36 AM
Screenshot 2023-09-13 at 10 40 47 AM

### CODE REVIEW

File: server/lib/shared.py [hunk 1]
```diff
@@ -19,22 +19,41 @@
...
-        result[dcid] = values[0]
+  # dcids to get default name for
+  default_name_dcids = []
+  # if there's a specified property to use for the name, try to find names using
+  # that prop first
+  if prop:
+    name_prop_response = fetch.property_values(dcids, prop)
+    for dcid in dcids:
+      for dcid, values in name_prop_response.items():
+        name_found = False
```
Comment (pradh) [inline server/lib/shared.py:41]: There two variables called dcid in the same scope?

Maybe you want:

```
for dcid in dcids:
  if resp.get(dcid, []):
    result[dcid] = resp[dcid][0]
  else:
     default_name_dcids.append(dcid)  
```

ditto, below

**Review by pradh — COMMENTED** [review]
Very nice!

**Review by dwnoble — APPROVED** [review]
Nice!

File: server/lib/shared.py (hunk 1 above)
Comment (chejennifer) [reply server/lib/shared.py]: updated

**Review by pradh — APPROVED** [review]

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**